astroid/context.py

```diff
@@ -164,10 +163,10 @@ def __init__(
     ):
         self.args = args  # Call positional arguments
         if keywords:
-            keywords = [(arg.arg, arg.value) for arg in keywords]
+            keywords_list = [(arg.arg, arg.value) for arg in keywords]
```

Comment:
I'm not a fan of putting the type in the variable name: it increase the mental load, you can check the type if you need it and the plural form indicate it's an iterable.

Reply:
I changed it to `keywords_tuples`. Open for any better suggestions.

Reply:
> I changed it to `keywords_tuples`. Open for any better suggestions.

I think that's worse, let's just leave it at `keywords`. It's assigned to `self.keywords` after all.

Reply:
I meant that ``keywords`` was fine :)

Reply:
`Mypy` will complain then. `keywords` is defined as `keywords: Optional[List["Keyword"]] = None,` on L161. That's why I introduced the name change.

Reply:
```python
keywords: List[Tuple[str, str]] = [(arg.arg, arg.value) for arg in keywords]
```

Will raise a `no-redef` warning..

Reply:
Hmm, maybe ``keywords_as_list`` then ?

Reply:
> Hmm, maybe `keywords_as_list` then ?

They are a list already. Maybe `keywords_list` wasn't that bad after all.

Reply:
Went with `arg_value_pairs` 😅 Is that okay?